Conversions/Lengthconversions.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class LengthConversions
+{
+    public static function mToKm($m)
+    {
+        if (!is_numeric($m)) {
+            throw new \InvalidArgumentException("Invalid input for mToKm: expected numeric, got string ('{$m}')");
+        }
+        return round($m / 1000, 6);
```

Comment:
Rather than using a magic number (1000) - I'd prefer to use a named class constant, e.g.

```
private const M_PER_KM = 1000.0;
private const KM_PER_MI = 1.609344;            // exact standard
private const IN_PER_CM = 0.39370078740157477;  // 1 / 2.54
```